lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -279,6 +280,7 @@ protected function assignDefaultVersionValue($entity, array $id)
      */
     protected function fetchVersionValue(VersionFieldMetadata $versionProperty, array $id)
     {
+        /** @var ClassMetadata */
```

Comment:
`@var` needs the variable name.

Reply:
Sure, but also I realise that `@var` should [only be used for properties](https://docs.phpdoc.org/references/phpdoc/tags/var.html), according to phpDoc (Psalm and PHPStorm support it, but that support is outside the spec).

Maybe I should replace with an `assert`?

Reply:
Wouldn't be pedantic about this one, it's generally well supported.

Reply:
ok!